revscoring/utilities/tests/test_union_merge_observations.py

```diff
@@ -0,0 +1,48 @@
+import json
+import os.path
+import tempfile
+
+import revscoring.utilities.union_merge_observations
+
+
+def test_union():
+    # Get test fixtures.
+    data_dir = os.path.dirname(__file__) + "/data"
+    in_files = [
+        data_dir + "/labeled_revisions.json",
+        data_dir + "/labeled_foo.json",
+    ]
+
+    # Receive the output in a temporary file.
+    (fh, out_file) = tempfile.mkstemp()
```

Comment:
It is an integration test :)  At some level, everything is an integration test--what we gain by writing a second test to only run union_merge() is a little granularity about where the failure is, if something goes wrong.

I would resist (to a reasonable point) taking out the integration test, because with the .extend(read_observations), a significant amount of the logic is taken out of union_merge.  That in itself is wonderful and I'm glad we did it, but I don't see why we would want to omit testing the extend?